src/logger/winston-logger.js

```diff
@@ -22,7 +22,8 @@ export function createProdLogger() {
   const opts = {
     // for stackdriver error reporting
     serviceContext: {
-      service: 'twreporter-react-server'
+      service: 'twreporter-react-server',
```

Comment:
Minor. Why do we want to have different service name  other than container name?

Reply:
How about `{cluster_name}:{container_name}:{namespace}:server` ?